crates/worktree/src/worktree.rs

```diff
@@ -4354,8 +4354,9 @@ impl BackgroundScanner {
                 let canonical_path = match self.fs.canonicalize(&child_abs_path).await {
                     Ok(path) => path,
                     Err(err) => {
-                        log::error!("error reading target of symlink {child_abs_path:?}: {err:#}",);
-                        continue;
+                        log::warn!("error reading target of symlink {child_abs_path:?}: {err:#}",);
```

Comment:
We can remove this and reduce the noise as we are falling back it doesn't help with anything. But just thought keeping it might be good for debugging in future for bugs related to symlink. happy to remove this if not deemed necessary.